airbyte-integrations/connectors/source-bamboo-hr/source_bamboo_hr/schemas/time_off_requests.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": true,
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
+    },
+    "employeeId": {
+      "type": ["null", "string"]
+    },
+    "name": {
+      "type": ["null", "string"]
+    },
+    "start": {
+      "type": ["null", "string"],
+      "default": ""
+    },
+    "end": {
+      "type": ["null", "string"],
+      "default": ""
+    },
+    "created": {
+      "type": ["null", "string"],
+      "default": ""
+    },
+    "status": {
+      "type": ["null", "object"]
```

Comment:
You miss this object here. Can you add the proerties and additionalProperties?

Reply:
thanks ! done